### PR TITLE
Updating Vector<T> documentation to include table of hardware acceler…

### PR DESCRIPTION
**Pull request by Braini01**
…ated operations

Fixes this issue from corefx repository: https://github.com/dotnet/corefx/issues/32184

## Summary

Just added the table basically

### CODE REVIEW

File: xml/System.Numerics/Vector`1.xml [hunk 1]
```diff
@@ -48,7 +48,20 @@
  <xref:System.Numerics.Vector%601> is an immutable structure that represents a single vector of a specified numeric type. The count of a <xref:System.Numerics.Vector%601> instance is fixed, but its upper limit is CPU-register dependent. It is intended to be used as a building block for vectorizing large algorithms.  
   
  The <xref:System.Numerics.Vector%601> structure provides support for hardware acceleration.  
-  
+ 
+ The following table shows which primitive numeric data type and operation combination uses intrinsic instructions for faster executions:
```
Comment (mairaw) [inline xml/System.Numerics/Vector`1.xml]: this table doesn't render well:
image

Maybe try using a MarkDown table instead?

| data type | + | - | \* | / |
|------------|---|--|--|--|
|sbyte|Yes|Yes|No|No|

For example:
```
| data type | + | - | \* | / |
|------------|---|--|--|--|
|sbyte|Yes|Yes|No|No|
```

Comment (mairaw) [reply xml/System.Numerics/Vector`1.xml]: Copied the full table from the corefx issue:
```
| | `+` | `-` | `*` | `/` |
| --- | :---: | :---: | :---: | :---: |
|`sbyte` | Yes | Yes | No | No |
|`byte` | Yes | Yes | No | No |
|`short` | Yes | Yes | Yes | No |
|`ushort` | Yes | Yes | No | No |
| `int` | Yes | Yes | Yes | No |
| `uint` | Yes | Yes | No | No |
| `long` | Yes | Yes | No | No |
| `ulong` | Yes | Yes | No | No |
| `float` | Yes | Yes | Yes | Yes |
| `double` | Yes | Yes | Yes | Yes |
```

| | `+` | `-` | `*` | `/` |
| --- | :---: | :---: | :---: | :---: |
|`sbyte` | Yes | Yes | No | No |
|`byte` | Yes | Yes | No | No |
|`short` | Yes | Yes | Yes | No |
|`ushort` | Yes | Yes | No | No |
| `int` | Yes | Yes | Yes | No |
| `uint` | Yes | Yes | No | No |
| `long` | Yes | Yes | No | No |
| `ulong` | Yes | Yes | No | No |
| `float` | Yes | Yes | Yes | Yes |
| `double` | Yes | Yes | Yes | Yes |

Comment (Braini01) [reply xml/System.Numerics/Vector`1.xml]: Seems I did something wrong when I copied it. I hope it is fixed now.

**Pull request merged**